README.md

```diff
@@ -79,36 +79,64 @@ make db-clean # clean all existing db entries
 
 ### CLI Tool
 
-submit-order: submit a fast transfer order to the deployed gateway contract specifying an amount in USDC and a recipient.
-This only supports transferring USDC from EVM -> Osmosis, as any pre/post transfer swaps are typically handled by the [Skip Go](https://docs.skip.build/go/general/getting-started) API.
+First, build and install the CLI from the project root directory:
 
 ```shell
 make build-cli
```

Comment:
maybe below this we could have the commands you need to move the solver into the users PATH